### PR TITLE
Make README more readable

### PR DESCRIPTION
**Pull request by Randomblock1**
Puts things like install directions in expandable dropdowns. Makes the README much easier to read while still keeping all the information intact

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,6 @@ Usage:
   -p, --poolkey arg    Pool Public Key (48 bytes)
   -f, --farmerkey arg  Farmer Public Key (48 bytes)
   -G, --tmptoggle      Alternate tmpdir/tmpdir2 (default = false)
-      --hwinfo         Print information regarding underlying hardware
```
Comment (madMAx43v3r) [inline README.md:33]: that was added lately, shouldn't be removed

**Comment by madMAx43v3r** [comment]
except the one line, it looks all good, thx

**Comment by Randomblock1** [comment]
I think I copied the README from the other PR before hwinfo was added, sorry

**Comment by madMAx43v3r** [comment]
yeah i figured, thx

**Pull request merged**